test/e2e/specs/plugins/plugins__browse.ts

```diff
@@ -103,6 +103,7 @@ describe( DataHelper.createSuiteTitle( 'Plugins: Browse' ), function () {
 			'Search Engine Optimization',
 			envVariables.VIEWPORT_NAME !== 'mobile' ? true : false
 		);
+		await page.waitForURL( new RegExp( `/plugins/browse/seo/${ siteUrl }$`, 'g' ) );
```

Comment:
**nit:non-blocking** - a small question, is the intention to make sure the site slug comes at the end?

In any case, the 'g' flag doesn't seem necessary from my experience. Have you run into faulty results without it?

Reply:
> is the intention to make sure the site slug comes at the end?

The intention was to make a more specific regex as the bug it was trying to solve was that it being redirected to `/plugins/browse/seo`. So in the past, if we had used `/plugins/browse/seo` as regex the test would still be passing because we didn't specify the URL must end with it.

In this case, as we are specifying `siteURL`, we don't have any case currently where it might fail, so let me know if you prefer to remove it.

> the 'g' flag doesn't seem necessary from my experience

That's right, I removed it https://github.com/Automattic/wp-calypso/pull/79803/commits/9266509680d4c9c7a81411f429c7c21c0b4de8b3